ttnn/ttnn/operations/conv/tt_py_composite_conv.py

```diff
@@ -205,7 +205,7 @@ def calculate_per_core_out_matrix_width_ntiles(logical_grid_y, override):
         f"PARALLEL CONFIG :: {is_1d_systolic} :: {input_channels} :: {output_channels} :: {sliding_window_op_params} :: {config_override} -> {num_cores_nhw} :: {grid_size} :: {per_core_out_matrix_height_ntiles} :: {per_core_out_matrix_width_ntiles}"
     )
 
-    return ttl.tensor.OptimizedConvParallelizationConfig(
+    return ttnn.operations.conv2d.OptimizedConvParallelizationConfig(
```

Comment:
this should be ttnn.conv2d, right?

Reply:
ttnn.conv2d is currently a function. This file is internal to ttnn, so I thought it would be better to use the fully qualified name.

Reply:
right, this is internal stuff not exposed to the user.